src/data-table/types.ts

```diff
@@ -9,13 +9,19 @@ export interface DataTableProps {
     /** css class to put on table **/
     className?: string | undefined;
     selectedIds?: string[] | undefined;
+    disabledIds?: string[] | undefined;
     /** enable checkboxes */
     enableMultiSelect?: boolean | undefined;
     /** display select all checkbox */
     enableSelectAll?: boolean | undefined;
+    /** display action bar */
+    enableSelectionBar?: boolean | undefined;
+    enableStickyHeader?: boolean | undefined;
+    selectionBarContent?: ReactNode | undefined;
```

Comment:
following the terminology used in figma


```suggestion
    enableActionBar?: boolean | undefined;
    enableStickyHeader?: boolean | undefined;
    actionBarContent?: ReactNode | undefined;
```